src/ilx/EraseClosures.fs

```diff
@@ -482,27 +481,18 @@ let rec convIlxClosureDef cenv encl (td: ILTypeDef) clo =
               let cloTypeDef = 
                 { Name = td.Name
                   GenericParams= td.GenericParams
-                  Access=td.Access
+                  Attributes = td.Attributes
                   Implements = List.empty
-                  IsAbstract = false
                   NestedTypes = emptyILTypeDefs
-                  IsSealed = true
-                  IsSerializable=td.IsSerializable 
-                  IsComInterop=false
-                  IsSpecialName=false
                   Layout=ILTypeDefLayout.Auto
-                  Encoding=ILDefaultPInvokeEncoding.Ansi
-                  InitSemantics=ILTypeInit.BeforeField
                   Extends= Some cenv.mkILTyFuncTy
                   Methods= mkILMethods ([ctorMethodDef] @ [nowApplyMethDef]) 
                   Fields= mkILFields (mkILCloFldDefs cenv nowFields)
                   CustomAttrs=emptyILCustomAttrs
                   MethodImpls=emptyILMethodImpls
                   Properties=emptyILProperties
                   Events=emptyILEvents
-                  HasSecurity=false 
-                  SecurityDecls=emptyILSecurityDecls 
-                  tdKind = ILTypeDefKind.Class}
+                  SecurityDecls=emptyILSecurityDecls }.WithSpecialName(false).WithImport(false).WithHasSecurity(false).WithAbstract(false).WithSealed(true).WithInitSemantics(ILTypeInit.BeforeField).WithEncoding(ILDefaultPInvokeEncoding.Ansi)
```

Comment:
same here. many new intermediate records

Reply:
Again, I don't think it matters too much.  

An alternative would be to have ``.WithFlags(specialName=false)`` and so on which would be fine by me.

That said, I thought ``false`` was the default for all of these?  So I'm not sure we need to call anything here?